github-dark.css

```diff
@@ -1091,7 +1090,9 @@
   table.files td.message a:hover, .link-gray:hover, .news .link-gray-dark:hover,
   .news a.link-gray-dark:hover, .reblame-link:hover,
   .orgs-help-item-octicon .octicon, span.btn-link, .btn-link, a.btn-link,
-  .merge-status-list .status-actions:hover {
+  .merge-status-list .status-actions:hover,
+  .tree-browser tr.navigation-focus td .octicon-chevron-right,
+  .tree-browser tr[aria-selected="true"] td .octicon-chevron-right, {
```

Comment:
Trailing comma here invalidates the entire block.

Reply:
fixed thanks ;) in between moving from stylus to git maybe it slipped into a coma :)